### PR TITLE
Update Screenshots for WebMethods.io Connector in 10.17+

### PR DESCRIPTION
**Pull request by MarkJMReynolds-SAG**
Also: Reduced image count by one.
Will need to be cherry-picked into 10.17 branch after approval, please let me know if you want me to drive.

### CODE REVIEW

**Comment by BeateRixen** [comment]
@MarkJMReynolds-SAG In the image appswitcher-integration the icon for MLW is shown. Since this will no longer be available with future releases I recommend to remove this from the image.
Can you influence the text in the salesforce image? It says Cumulocity but our product is called Cumulocity IoT.

**Comment by MarkJMReynolds-SAG** [comment]
@BeateRixen Sure, no problem. Have updated the images as required.

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@MarkJMReynolds-SAG Could you please have this approved by someone from your team as well?

File: content/users-guide/webMethods.md [hunk 1]
```diff
@@ -44,4 +43,4 @@ WebMethods.io also provides pre-configured workflows which are called "recipes".
 
 ![webMethods.io Example Recipe](/images/users-guide/webMethods.io/wmio-recipe-salesforce.png)
 
-More examples and technical guides can be found on the [{{< sag-dev-community >}} website](https://techcommunity.softwareag.com/en_en/webmethods-apis-integration-microservices.html).
+More examples and technical guides can be found on the [{{< sag-dev-community >}} website](https://techcommunity.softwareag.com/en_en/webmethods-apis-integration-microservices.html). In particular, for more detailed information about the Cumulocity IoT triggers and connectors in webMethods.io please refer to [these articles]({{< link-sag-tech-forum >}}tag/webmethods-c8y-connector).
```
Comment (BeateRixen) [inline content/users-guide/webMethods.md:46]:
```suggestion
More examples and technical guides can be found on the [{{< sag-dev-community >}} website](https://techcommunity.softwareag.com/en_en/webmethods-apis-integration-microservices.html). In particular, for more detailed information about the {{< product-c8y-iot >}} triggers and connectors in webMethods.io refer to [these articles]({{< link-sag-tech-forum >}}tag/webmethods-c8y-connector).
```

**Review by BeateRixen — APPROVED** [review]

**Review by thomaswinkler — APPROVED** [review]

**Pull request merged**